biolink-model.yaml

```diff
@@ -2927,7 +2948,7 @@ classes:
       - DOID
       - OMIM
       - ORPHANET
-      - ORPHA
+      # - ORPHA  # no idea what this prefix resolves to globally. Is is an abbrevation for ORPHANET? not used in model
```

Comment:
https://github.com/monarch-initiative/dipper/blob/master/dipper/curie_map.yaml#L81

'ORPHA': 'http://www.orpha.net/ORDO/Orphanet_'                      # Rare diseases and Orphan drugs

------------------------------
was originally 'ORPHANET' (2015) was changer to be case sensitive 'Orphanet' with

https://github.com/monarch-initiative/dipper/issues/73

then  to 'ORPHA' 

https://github.com/monarch-initiative/dipper/issues/469

My understating is 'ORPHA' is the prefix the people who did the work prefer

Reply:
Thanks @TomConlin.

Now... I assume that I should:

1. Resolve to this unitary prefix (delete ORPHANET; standardise on ORPHA)
2. Rewrite all ORPHANET prefixes with ORPHA

Will the elimination of the ORPHANET prefix from the model trigger any issues anyway? (or, do I care, LOL?)